andela/ah-baqi-frontend
# #166980197 Hound Configuration

### What does this PR do?
- Add hounds integration
### Description of Task to be completed?
- [x] Ensure PRs follow ES6 Linting standards
- [x] Comment on lines that fail to observe linting for fixing
### How should this be manually tested?
- Access the repository on github from here
- Switch to the branch ch-configure-hound-166980197
- Comments from hound should be visible
### Any background context you want to provide?
- None

### What are the relevant pivotal tracker stories?
[#166980197](https://www.pivotaltracker.com/story/show/166980197)
